internal/pkg/webconfig/webconfig.go

```diff
@@ -232,12 +232,14 @@ func (s *Server) Run(ctx context.Context) ([]byte, error) {
 		s.Stop()
 	}()
 
-	fmt.Println("Webconfig started.  You may POST a configuration to:")
+	s.logger.Println("Webconfig started. You may POST a configuration to:")
 
 	for _, ip := range ips {
-		fmt.Printf("\t%s\n", ip.String())
+		s.logger.Printf("\thttps://%s\n", ip.String())
 	}
 
+	s.logger.Println("You may also visit any of the above in your browser.")
+
```

Comment:
I wasn't seeing this printed out in the console on a real machine. The logger is configured to write to the correct place regardless of which mode we are running.

Reply:
Curious; I used the fmt method because _I_ wasn't seeing the logs with the _logger_.,

Reply:
Hmm. Well that is strange. Maybe something broken specifically around the logger and container mode. Regardless, I think this is the intended way, and it should work. Sounds like a bug outside of this.